src/index.html

```diff
@@ -14,10 +14,9 @@
       <h1>what's cookin'?</h1>
       <input class="input" type="text" id ="searchInput" placeholder="Search recipes...">
       <button id ="searchRecipes">search</button>
-      <button class="home-button hidden">home</button>
     </div>
     <div class="nav-div flex-row">
-      <button class="hidden">home</button>
+      <button class="home-button hidden">home</button>
```

Comment:
We'd previously created a home button when we already had one. So we corrected the mistake we made.